### DISCUSSION TITLE
consider go-linter-runner project

### DISCUSSION THREAD

**Discussion by ccoVeille** [Antonboom/testifylint#187] [Ideas]
I discovered this project yesterday

https://github.com/alingse/go-linter-runner (hi  @alingse  :wave: )

 @alingse popped out an issue on a repository I'm watching
https://github.com/montanaflynn/stats/issues/84

@Antonboom I know you often look for regression/changes when working on testifylint

Examples:
- https://github.com/Antonboom/testifylint/issues/93#issuecomment-2133226047
- https://github.com/Antonboom/testifylint/pull/97#issuecomment-2133928388

is this something that could be useful for you ?

are you using such a tool by yourself? or are you launching your tests on local repository like I do.

cc @mmorel-35 who might be interested

**Comment by alingse** [comment]
thx for recommand. I write a testifylint.yaml job , and run the actions for top 1k github golang repo.
you can check the output  https://github.com/alingse/go-linter-runner/issues/2

if you only care about the panic, I can update the `includes` option, you can fork or modify the repo if you would like.

**Comment by Antonboom** [comment]
hi!

sorry, I did not catch how to use the project and what actions are expected from me as an author

**Comment by ccoVeille** [comment]
> hi!
> 
> sorry, I did not catch how to use the project and what actions are expected from me as an author

The tool made by @alingse can be used to launch testifylint on thousands of go projects.

So you would be able to see how things varies from one version to another, even let's say code that is currently is PR and development

**Comment by ccoVeille** [comment]
But maybe https://github.com/alingse/go-linter-runner/issues/4 needs to be addressed before we could use it the way I think

**Comment by ccoVeille** [comment]
The result are starting to flow see https://github.com/alingse/go-linter-runner/issues/2 

I created this for a bigger readability https://github.com/Antonboom/testifylint/discussions/159
